examples/foundry/Makefile

```diff
@@ -0,0 +1,17 @@
+
+all: build test
+
+build:
+	forge install foundry-rs/forge-std --no-commit --no-git
+	forge soldeer install @oasisprotocol-sapphire-contracts~0.2.11 --config-location foundry
```

Comment:
Could we simply put a `dependencies/oasisprotocol-sapphire-contracts` symlink pointing to `../../../contracts` folder instead? This way we could always test whether sapphire contracts broke any foundry example specifics.

Reply:
Yes, this is better. Added symlink to ../../../contracts/contracts.